app/models/state_file_base_intake.rb

```diff
@@ -8,6 +8,12 @@ class StateFileBaseIntake < ApplicationRecord
   has_many :dependents, -> { order(created_at: :asc) }, as: :intake, class_name: 'StateFileDependent', inverse_of: :intake, dependent: :destroy
   has_many :efile_submissions, -> { order(created_at: :asc) }, as: :data_source, class_name: 'EfileSubmission', inverse_of: :data_source, dependent: :destroy
 
+  delegate :tax_return_year, to: :direct_file_data
+
+  def direct_file_data
```

Comment:
Should we worry about lifecycle issues with `||=`? My hunch is it's better not to bother with `@... ||=` and instead always return `DirectFileData.new(raw_direct_file_data)`.

Not a blocker.